common/views/components/Control/index.tsx

```diff
@@ -192,7 +192,7 @@ const BaseControl: ForwardRefRenderFunction<HTMLButtonElement, Props> = (
     'aria-expanded': ariaExpanded,
     'aria-pressed': ariaPressed,
     'data-gtm-trigger': dataGtmTrigger,
-    'tab-index': tabIndex,
```

Comment:
'tab-index' does not exist; tabIndex (JSX/react) will be transformed into [`tabindex`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex).